src/Libm.jl

```diff
@@ -1,5 +1,80 @@
 module Libm
 
-# package code goes here
+export xexp
 
-end # module
+const PI4_A = 0.78539816290140151978
+const PI4_B = 4.9604678871439933374e-10
+const PI4_C = 1.1258708853173288931e-18
+const PI4_D = 1.7607799325916000908e-27
+
+const M_4_PI = 1.273239544735162542821171882678754627704620361328125
+
+const L2U = .69314718055966295651160180568695068359375
+const L2L = .28235290563031577122588448175013436025525412068e-12
+const R_LN2 = 1.442695040888963407359924681001892137426645954152985934135449406931
+
+@inline double_to_raw_long_bits(d::Float64) = reinterpret(Int64, d)
+
+@inline long_bits_to_double(i::Int64) = reinterpret(Float64, i)
+
+@inline xfabs(x::Float64) = long_bits_to_double((0x7fffffffffffffff % Int64) & double_to_raw_long_bits(x))
+
+@inline function mulsign(x::Float64, y::Float64)
+  return long_bits_to_double(double_to_raw_long_bits(x)^(double_to_raw_long_bits(y) & (1 % Int64 )<< 63))
+end
+
+@inline sign(d::Float64) =  mulsign(1.0, d)
+@inline mla(x::Number, y::Number, z::Number) = x * y + z
```

Comment:
Does this need to be expanded to `x*y+z` or can this be a fused multiply add? Julia's `muladd` function might be helpful, but I am unsure about the consequences for precision.